Library/Formula/bugzilla.rb

```diff
@@ -0,0 +1,22 @@
+class Bugzilla < Formula
+  desc "Bugzilla is server software designed to help you manage software development."
+  homepage "https://www.bugzilla.org"
+  url "https://ftp.mozilla.org/pub/mozilla.org/webtools/bugzilla-5.0.1.tar.gz"
+  version "5.0.1"
+  sha256 "f0f85bf405984770d65f60f5a828ea4d173acaca4aa85e4e58b4e88b1ae186a5"
+
+  depends_on :x11 
```

Comment:
If this dependency can't be made optional, then I'm afraid it'll need to go in our [homebrew/x11](https://github.com/Homebrew/homebrew-x11) tap.